homeassistant/components/apsystems_local/__init__.py

```diff
@@ -0,0 +1,34 @@
+"""The APsystems local API integration."""
+
+from __future__ import annotations
+
+import logging
+
+from APsystemsEZ1 import APsystemsEZ1M
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_IP_ADDRESS, Platform
+from homeassistant.core import HomeAssistant
+
+from .coordinator import ApSystemsDataCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up this integration using UI."""
+    entry.runtime_data = {}
+    api = APsystemsEZ1M(ip_address=entry.data[CONF_IP_ADDRESS], timeout=8)
+    coordinator = ApSystemsDataCoordinator(hass, api)
+    await coordinator.async_config_entry_first_refresh()
+    entry.runtime_data = {**entry.data, "COORDINATOR": coordinator}
```

Comment:
No need to store `entry.data` in the config entry. 
```suggestion
    entry.runtime_data = coordinator
```